src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -7,6 +7,7 @@
 	 */
 	public boolean isOddNumber(int a){
 		int tmp = (a-1)%2;
+                int foo = 0;
 		
 		if ( tmp != 0 ){ // Fix : tmp == 0
```

Comment:
This line (12) has been identified with a suspiciousness value of 53.88%.
The following failing tests covered this line: `nopol_examples.nopol_example_3.NopolExampleTest#test3`,`nopol_examples.nopol_example_3.NopolExampleTest#test4`,`nopol_examples.nopol_example_3.NopolExampleTest#test1`,`nopol_examples.nopol_example_3.NopolExampleTest#test2`,`nopol_examples.nopol_example_3.NopolExampleTest#test7`,`nopol_examples.nopol_example_3.NopolExampleTest#test8`,`nopol_examples.nopol_example_3.NopolExampleTest#test5`,`nopol_examples.nopol_example_3.NopolExampleTest#test6`,`nopol_examples.nopol_example_3.NopolExampleTest#test9`